src/unstract/adapters/x2text/llama_parse/src/static/json_schema.json

```diff
@@ -0,0 +1,52 @@
+{
+    "title": "Llama Parse X2Text",
+    "type": "object",
+    "required": [
+      "api_key"
+    ],
+    "properties": {
+      "adapter_name": {
+        "type": "string",
+        "title": "Llama Parse Adapter ID",
+        "default": "",
+        "description": "Provide a unique name for this adapter instance. Example: Llama parse1"
+      },
+        "api_key": {
+            "type": "string",
+            "title": "API Key",
+            "format": "password",
+            "default": "",
+            "description": "Provide the token (API Key) of the Llama Parse server"
+        },
+      "url": {
+        "type": "string",
+        "title": "URL",
+        "format": "url",
+        "default": "https://api.cloud.llamaindex.ai",
+        "description": "Provide the Base URL of llama Parse server."
+      },
+      "result_type": {
+        "type": "string",
+        "title": "Result Type",
+        "enum": [
+          "text",
+          "markdown"
+        ],
+        "default": "text",
+        "description": "Choose the type of result. Markdown or text."
+      },
+      "num_workers": {
+        "type": "integer",
+        "title": "Number of Workers",
+        "default": 4,
+        "description": "Used for multiple files to split the API calls between the workers."
+      },
```

Comment:
Since we only work with single files per call - I don't think this would be helpful to the user. In fact it would only confuse them more. I suggest removing it